src/main/java/com/softserve/paymentservice/service/AmountCalculator.java

```diff
@@ -0,0 +1,33 @@
+package com.softserve.paymentservice.service;
+
+import com.softserve.paymentservice.dto.PaymentInfoDto;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AmountCalculator {
+
+    private int amount;
+
+    public enum Tariffs {
```

Comment:
It seems like this enum isn't tightly related to this class. Shouldn't it be extracted?